src/views/notifications/OrderStatusNotification.tsx

```diff
@@ -85,12 +88,24 @@ export const OrderStatusNotification = ({
       if (localOrder.errorStringKey) {
         orderStatusStringKey = STRING_KEYS.ERROR;
         orderStatusIcon = <$WarningIcon iconName={IconName.Warning} />;
-        customContent = <span>{stringGetter({ key: localOrder.errorStringKey })}</span>;
+        customContent = (
+          <span>
+            {stringGetter({
+              key: localOrder.errorStringKey,
+              params: {
+                EQUITY_TIER_LEARN_MORE: (
+                  <Link href={equityTiersLearnMore} onClick={(e) => e.stopPropagation()}>
+                    {stringGetter({ key: STRING_KEYS.LEARN_MORE })} →
```

Comment:
nit, can we inline this and make it text-color-1? (so it matches a lil better). I'm also a mild advocate for using LEARN_MORE_ARROW instead of manually having to append an arrow here!